rust/src/address.rs

```diff
@@ -302,44 +302,95 @@ impl JsonSchema for Address {
     fn is_referenceable() -> bool { String::is_referenceable() }
 }
 
+#[wasm_bindgen]
+#[derive(Clone, Debug, Eq, Ord, PartialEq, PartialOrd)]
```

Comment:
It might be good to leave a comment here though that you shouldn't == against the header and should instead & or else the network ID could get in the way.

Reply:
Yeah you're right, this felt like a footgun so I changed the way this works a bit